pyneuroml/plot/PlotMorphologyVispy.py

```diff
@@ -769,6 +871,10 @@ def plot_3D_cell_morphology(
         axon_segs = cell.get_all_segments_in_group("axon_group")
     except Exception:
         axon_segs = []
+    
+    if isPCA:
+        cell = PCA_transformation(cell)
+    current_canvas = current_view = None
 
     if current_canvas is None or current_view is None:
         view_min, view_max = get_cell_bound_box(cell)
```

Comment:
I guess we should also set the view to origin etc. here as we do for `plot_interactive_3d`?